src/components/StatusUpdater/index.tsx

```diff
@@ -114,7 +115,15 @@ const StatusUpdaterProtected = ({ activity, current, fullWidth, user, thisOrg }:
     setConfirming(true);
   }
 
-  const actions = getStatusOptions(current, activity.startTime, activity.forceStandbyOnly, activity.earlySignInWindow);
+  const lastOrgId = participant?.timeline[0].organizationId;
+  const currentOrgId = participant?.organizationId;
+  let actions;
+  if (isActive(current) && currentOrgId !== lastOrgId) {
+    const lastOrgName = activity.organizations[lastOrgId].rosterName ?? activity.organizations[lastOrgId].title;
+    actions = [{ id: 7, newStatus: ParticipantStatus.SignedOut, text: `Sign Out from ${lastOrgName}` }];
```

Comment:
Magic number 7 for the ID. Should that be an enum value?

Reply:
fixed